src/Mover.ts

```diff
@@ -763,6 +799,90 @@ export class MoverAPI implements Types.MoverAPI {
         }
     };
 
+    private _focusMoverFor(
```

Comment:
Why is this feature necessary?
 
- I want arrow navigation inside a DOM element - I apply mover
- I press an arrow key outside the mover
- focus should go into the mover

Why does this become a platform problem?

Reply:
We're not perfect yet and allow focus go to <body> from time to time. The current version of Teams has this helper that if you're seeing a chat, and you press arrow keys, your focus will be moved from <body> to the chat regiving you a meaningful starting point (otherwise you would need to Tab through the entire app again and that's not the best experience). So, it will be possible to just set `nextFor: 'body'` on the chat's mover and reuse Tabster's machinery to handle that case.

I don't think it's a part of some standard but as far as I'm aware of, there are multiple people (including the screen reader users) who find this behaviour very helpful. It is easy to implement on the Tabster level and it is guaranteed to be consistently compatible with all our focus management primitives, giving more stability comparing to the application level implementation.

Another possible use case is you might be inside an input, but you might want to move focus to some choices selection list below or above. That scenario will also be solvable with one Tabster property.